recipes.txt

```diff
@@ -16,6 +16,8 @@ ffmpeg -i input_file -map 0 -dn -c:v ffv1 -level 3 -g 1 -slicecrc 1 -slices 16 -
 ffmpeg -i concat:input_file_1\|input_file_2\|input_file_3 -c:v libx264 -c:a aac output_file.mp4
 # Transcode to an H.265/HEVC MP4
 ffmpeg -i input_file -c:v libx265 -pix_fmt yuv420p -c:a copy output_file
+# Transcode to H.264/H.265 using the GPU
+ffmpeg -i input_file -c:v h264_nvenc -preset llhq -rc:v vbr_hq -cq:v 19 -b:v 8000k -maxrate:v 12000k -profile:v high -c:a copy output_file
```

Comment:
You could add both recipes the one for H.264 and the one for H.265. That way both can be imported easily into FFCommand Engine.